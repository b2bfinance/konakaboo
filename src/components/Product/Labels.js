import React from 'react';
import styled from 'styled-components';
import Label from '../Label';
import { queries } from '../../utils/media';

export const Wrapper = styled.div`
  display: flex;
  flex-wrap: wrap;
  line-height: 1;
  margin: 8px 0;

  ${queries.desktop`
    margin: 0;
  `}
`;

export default ({ labels }) => (
  <Wrapper>
    {labels.map((label, i) => (
      <Label key={i} label={label} />
    ))}
  </Wrapper>
);
