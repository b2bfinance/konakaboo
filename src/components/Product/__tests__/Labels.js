import React from 'react';
import { render } from 'enzyme';
import Labels from '../Labels';

describe('Labels', () => {
  test('renders correctly', () => {
    expect(render(<Labels labels={['foo', 'bar']} />)).toMatchSnapshot();
  });
});
