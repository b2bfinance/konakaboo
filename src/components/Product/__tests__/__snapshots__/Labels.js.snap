// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Labels renders correctly 1`] = `
.c1 {
  border: 1px solid;
  border-radius: 1rem;
  font-size: 80%;
  padding: 0.2rem 0.75rem;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c0:last-child div {
  margin-right: 0;
}

.c0 div {
  margin: .5rem .5rem 0 0;
}

@media (min-width:36em) {
  .c0 {
    margin-left: auto;
  }

  .c0 div {
    margin: 0 .5rem 0 0;
  }
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    foo
  </div>
  <div
    class="c1"
  >
    bar
  </div>
</div>
`;
